next-cloudinary/src/components/CldOgImage/CldOgImage.tsx

```diff
@@ -24,7 +24,8 @@ const CldOgImage = ({ excludeTags = [] as string[], twitterTitle, ...props }) =>
   // We need to include the tags within the Next.js Head component rather than
   // direcly adding them inside of the Head otherwise we get unexpected results
 
-  return (
+    return (
+    // @ts-ignore
```

Comment:
```
TS2741: Property 'children' is missing in type '{}' but required in type '{ children: React.ReactNode; }'.
```

Didn't fix this in this PR because I'm not really sure _how_ it should be addressed, and didn't want to bloat the PR by doing so